### PR TITLE
Add socials

### PR DESCRIPTION
**Pull request by DonaldKLee**
Added 1st year rep to the website
Hyperlinked each member with their socials

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -56,322 +56,383 @@ const Links = styled.div`
...
+      color: '#01DACC',
+      title: 'HackCamp Logistics Coordinator',
+      social: ""
+    },
+    {
+      img: '/assets/profiles/Oliver_Luo.png',
+      name: 'Oliver Luo',
+      emoji: '👴',
+      color: '#FFA500',
+      title: 'HackCamp Logistics Coordinator',
```
Comment (michelleykim) [inline components/Footer.js:368]: Can you change Oliver's role to Sponsorship Coordinator

**Review by michelleykim — CHANGES_REQUESTED** [review]
the changes lgtm, left a comment for a minor change. I wonder why the deploy has failed

File: components/Footer.js (hunk 1 above)
Comment (DonaldKLee) [reply components/Footer.js]: done!

**Comment by DonaldKLee** [comment]
> the changes lgtm, left a comment for a minor change. I wonder why the deploy has failed- would fixing eslint fix it?

@naijwu @meleongg do yk why the deploy is failing? seems to be a firebase issue

**Comment by meleongg** [comment]
Try running yarn lint locally, it looks like there's a React child key error

also can you add my social LOLLL I thought I updated it xd

image

File: components/Team.js [hunk 2]
```diff
@@ -122,24 +122,28 @@ export default function Team({ profiles }) {
...
-              key={i}
-              src={profile.img}
-              color={profile.color}
-              onClick={() => setSelectedProfile(profile)}
-              onMouseEnter={() => setSelectedProfile(profile)}
-              onMouseLeave={() => setSelectedProfile({})}
-            />
+            <a href={profile.social}>
+              <ProfileImage
+                key={i}
```
Comment (meleongg) [inline components/Team.js:139]: I think there are keys that aren't unique since the children in the above map could have the same key

Comment (DonaldKLee) [reply components/Team.js]: fixed

**Comment by meleongg** [comment]
There's an unused variable causing lint errors. Does `yarn build` work when you run locally - it should throw linting errors

image

image

**Comment by DonaldKLee** [comment]
> There's an unused variable causing lint errors. Does `yarn build` work when you run locally - it should throw linting errors
> 
> image image

fixed!

**Comment by meleongg** [comment]
@michelleykim should we also turn the "join the team" button off with this pr?

image

**Comment by michelleykim** [comment]
> @michelleykim should we also turn the "join the team" button off with this pr?
> 
> image

ooh yeah thatd be nice @DonaldKLee can u change this too lol luv u

**Comment by DonaldKLee** [comment]
> > @michelleykim should we also turn the "join the team" button off with this pr?
> > image
> 
> ooh yeah thatd be nice @DonaldKLee can u change this too lol luv u

done! did not need to touch the code for this. this is a firebase flag

**Review by michelleykim — APPROVED** [review]
lgtm!!

**Pull request merged**